src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2247,6 +2256,11 @@ private BoundExpression BindArgumentExpression(DiagnosticBag diagnostics, Expres
 
                     arguments[arg] = ((OutVarLocalPendingInference)argument).SetInferredType(parameterType, success: !hasErrors);
                 }
+                else if (argument.Kind == BoundKind.DefaultPendingInference)
+                {
+                    TypeSymbol parameterType = GetCorrespondingParameterType(ref result, parameters, arg);
+                    arguments[arg] = ((BoundDefaultPendingInference)argument).SetInferredType(parameterType);
+                }
                 else if (argument.Kind == BoundKind.OutDeconstructVarPendingInference)
```

Comment:
This is the wrong place to handle this. It should be handled as a "conversion from expression", like `null`.